src/mjolnir/shortcutbuilder.cc

```diff
@@ -337,18 +340,18 @@ bool IsEnteringEdgeOfContractedNode(GraphReader& reader, const GraphId& nodeid,
 }
 
 // Add shortcut edges (if they should exist) from the specified node
-uint32_t AddShortcutEdges(GraphReader& reader,
-                          const graph_tile_ptr& tile,
-                          GraphTileBuilder& tilebuilder,
-                          const GraphId& start_node,
-                          const uint32_t edge_index,
-                          const uint32_t edge_count,
-                          std::unordered_map<uint32_t, uint32_t>& shortcuts) {
+shortcut_stats AddShortcutEdges(GraphReader& reader,
+                                const graph_tile_ptr& tile,
+                                GraphTileBuilder& tilebuilder,
+                                const GraphId& start_node,
+                                const uint32_t edge_index,
+                                const uint32_t edge_count,
+                                std::unordered_map<uint32_t, uint32_t>& shortcuts) {
   // Shortcut edges have to start at a node that is not contracted - return if
   // this node can be contracted.
   EdgePairs edgepairs;
   if (CanContract(reader, tile, start_node, edgepairs)) {
-    return 0;
+    return {0UL, 0UL};
```

Comment:
nit UL is probably 64bits wide, ie on 64 bit architectures. you could use just U I'd you wanted but if we wanted to be technically correct for all architectures you should use the exact type

Reply:
I'll just do what the vast majority of valhalla is doing, passing ints:) what's ULL then on 64 bit archs?

Reply:
changed it to `return {uint32_t(0), uint32_t(0)};` instead